packages/expo-cli/src/commands/client/index.js

```diff
@@ -64,12 +64,24 @@ export default program => {
       const table = new CliTable({ head: ['Name', 'Identifier'], style: { head: ['cyan'] } });
       table.push(...devices.map(device => [device.name, device.deviceNumber]));
       log(table.toString());
-      const { addUdid } = await prompt({
-        name: 'addUdid',
-        message: 'Would you like to register new devices to use the Expo Client with?',
-        type: 'confirm',
-        default: true,
-      });
+
+      const promptToRegisterUdid = async () => {
+        const { addUdid } = await prompt({
+          name: 'addUdid',
+          message: 'Would you like to register new devices to use the Expo Client with?',
```

Comment:
If you have no devices registered this is a trick question 😁

Since there is just one possible answer why don’t just skip this prompt in that case?

Reply:
Maybe we should exit when a user answers "no" three times in a row and print an error "you should buy an iPhone" :D

Reply:
lol these comments had me dying 😂